h/accounts/schemas.py

```diff
@@ -404,5 +405,34 @@ class NotificationsSchema(CSRFSchema):
     )
 
 
+class CreateUserAPISchema(JSONSchema):
+    """Validate a user JSON object."""
+
+    schema = {
+        'type': 'object',
+        'properties': {
+            'authority': {
+                'type': 'string',
+                'format': 'hostname',
+            },
+            'username': {
+                'type': 'string',
+                'minLength': 3,
```

Comment:
At least until we get round to moving the actual schemas into data files, I reckon it would probably be wise to use `USERNAME_MIN_LENGTH` and `USERNAME_MAX_LENGTH` here. I think they're already available in this module.